libqtile/utils.py

```diff
@@ -200,6 +201,19 @@ def get_cache_dir() -> str:
     return cache_directory
 
 
+def get_config_file():
+    config_home = os.getenv("XDG_CONFIG_HOME", "~/.config")
+    if (config_file := Path(f"{config_home}/qtile/config.py").expanduser()).exists():
+        return config_file
+
+    xdg_config_dirs = os.getenv("XDG_CONFIG_DIRS", "/etc/xdg/").split(":")
+    for config_dir in xdg_config_dirs:
+        if (system_wide_config := Path(f"{config_dir}/qtile/config.py")).expanduser().exists():
+            return system_wide_config
```

Comment:
This is getting hard to read with all the parentheses, could you split out the assignments both here and on line 206? In fact, you've misplaced them, and here we could be returning a path without the resolved tilde ;)
```suggestion
    for config_dir in xdg_config_dirs:
        system_wide_config = Path(f"{config_dir}/qtile/config.py")).expanduser()
        if system_wide_config.exists():
            return system_wide_config
```

Reply:
done here: https://github.com/qtile/qtile/pull/4490/commits/e3a9eafe47703a71fdfb3bc0c6a60596dc64d5bf